apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.ejs

```diff
@@ -0,0 +1,269 @@
+											<form name="edit-override-form" data-user-id="<%= policy.student_uid %>" data-group-id="<%= policy.group_id %>" method="POST">
+												<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+												<!-- <button type="button" class="btn edit-override-button" id="edit-override-button" data-assessment-id="<%= policy.assessment_id %>" data-user-id="<%= policy.student_uid %>" data-group-id="<%= policy.group_id %>" data-credit="<%= policy.credit %>" data-start-date="<%= policy.start_date %>" data-end-date="<%= policy.end_date %>" data-created-by="<%= policy.created_by %>" data-note="<%= policy.note %>" data-type="<%= policy.type %>">
+													<i class="fas fa-edit"></i> Edit -->
+												<!-- </button> -->
+												<button type="button" class="btn edit-override-button" data-assessment-id="<%= policy.assessment_id %>" data-user-id="<%= policy.student_uid %>" data-group-id="<%= policy.group_id %>" data-credit="<%= policy.credit %>" data-start-date="<%= policy.start_date %>" data-end-date="<%= policy.end_date %>" data-created-by="<%= policy.created_by %>" data-note="<%= policy.note %>" data-type="<%= policy.type %>">
+													<i class="fas fa-edit"></i> Edit
+												</button>												
+											</form>
+											<form name="delete-override-form" method="POST">
+												<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+												<input type="hidden" name="__action" value="delete_override">
+												<input type="hidden" name="student_uid" value="<%= policy.student_uid %>">
+												<input type="hidden" name="assessment_id" value="<%= policy.assessment_id %>"> 
+  												<input type="hidden" name="group_name" value="<%= policy.group_name %>">
+												<!-- <button type="submit" class="btn delete-button"> -->
+													<button type="button" class="btn delete-button" data-toggle="modal" data-target="#deleteModal" data-student-uid="<%= policy.student_uid %>" data-assessment-id="<%= policy.assessment_id %>" data-group-id="<%= policy.group_id %>">
+													<i class="fas fa-remove"></i> Delete
+												</button>
+											</form>
+
+										</div>
+									</div>
+								</td>
+							</tr>
+						<% }); %>
+					</tbody>
+				</table>
+			</div>
+			<div class="card-footer">
+			</div>
+			<!-- Modal dialog for delete confirmation -->
+			<div class="modal fade" id="deleteModal" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel" aria-hidden="true">
+				<div class="modal-dialog" role="document">
+					<div class="modal-content">
+						<div class="modal-header">
+							<h5 class="modal-title" id="deleteModalLabel">Confirm Delete</h5>
+							<button type="button" class="close" data-dismiss="modal" aria-label="Close">
+								<span aria-hidden="true">&times;</span>
+							</button>
+						</div>
+						<div class="modal-body">
+							<p>Are you sure you want to delete this Access Override? </p>
```

Comment:
"Access Override" shouldn't be capitalized; ditto for anywhere else on this page.